metric-diagnostics/kubecost-prometheus-healthcheck.sh

```diff
@@ -53,7 +55,19 @@ check_metric_missing()
 
 check_metric_labels()
 {
-echo $1 $(curl -sG --data-urlencode "query=topk(1,$1)" "$HOST_NAME")
+    my_json=$(curl -sG --data-urlencode "query=topk(1,$1)" "$HOST_NAME")
```

Comment:
It may be better to check a metric that Kubecost uses like `container_cpu_allocation` as opposed to `topk`?

Reply:
$1 is the metric, topk is just looking at the first result only.